app/assets/stylesheets/spotlight/_pages.scss

```diff
@@ -267,4 +267,5 @@
 
 .zpr-link {
   @extend .btn-sm;
+  margin-top: 0.75rem;
```

Comment:
Ah, I guess we can't use a `mt-` thing here because there isn't one that would end up being that.  Would `spacer * .75` be appropriate here?

Reply:
Sure that makes sense. I've made the change.